src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -102,6 +105,21 @@ public void addMessage(Message message) {
     return messagesInConversation;
   }
 
+  /** Access the current set of Messages sent by a specific user. */
+  public Queue<Message> getMessagesByOwner(UUID authorid) {
+
+    Queue<Message> messagesByOwner = new PriorityQueue<>(messages.size(), 
```

Comment:
I'd change the name to authorMessages. Messages by owner seems to apply it is a hashmap indexed by owner